### ISSUE TITLE
Cannot edit post with sub-folder path - GetDocument failed: Unable to fetch

### ISSUE THREAD

**Issue by ylahssini** [tinacms/tinacms#2935]
## Description

Hi, I create a new post with sub-folder (cooking/pizza.md) and it generated in content folder, but i can't edited because it shows me an error, it says that the pizza.md file doesn't exist

### Steps to reproduce

1. Create a new NextJS app - `yarn create next-app`
2. Bootstrap with Tina - `npx @tinacms/cli@latest init`
3. Launch the app - `yarn dev`
4. Go to `http://localhost:3000/admin`
5. Add new post with document name `cooking/pizza` and save it
![image](https://user-images.githubusercontent.com/58909647/170379516-9a4d0df3-84ef-4409-b0c8-60d192c18c5c.png)
6. Click on vertical dots and click on edit button, The error show up:
![image](https://user-images.githubusercontent.com/58909647/170379945-33cca22e-2431-44f8-981d-03acbe9b3a42.png)

### Expected result

It must go to edit page

### Actual result

The error show up:
`[Error] GetDocument failed: Unable to fetch, errors:`
`ENOENT: no such file or directory, open '/Users/joedoe/Documents/next-tina/content/posts/pizza.md'`

#### Tinacms Package Versions

```text

```

#### Tools

| Tool    | Version |
| ------- | ------- |
| browser | Firefox        |
| os      |  Mac os 12.2.1       |
| node    | 14.17.6        |
| npm     |  6.14.15       |

**Comment by kldavis4** [comment]
Discord user @Brian Barrow is reporting a similar / related issue.

He has a collection at path `content/post` with nested subfolders organized by year and date. So an example post might be `content/post/2022/05/test_post.md`.

Posts are rendering in the CMS as:

<img width="334" alt="Screen_Shot_2022-06-02_at_11 53 35_AM" src="https://user-images.githubusercontent.com/370955/171728331-f3766c67-5b7e-4390-be0d-394e583bc268.png">

But the links are not working make the content uneditable.

**Comment by brandlewco** [comment]
I still encounter this error. I created test content in pages like so "subfolder/item"
But it searches for it in the root of pages, even though the content was successfully added at pages/subfolder/item.md


```
Error: Unable to fetch, please see our FAQ for more information: https://tina.io/docs/errors/faq/
  
        Errors: 
	Unable to find record content/pages/item.md
```

**Comment by storynory** [comment]
Me too!

**Comment by storynory** [comment]
I've realised that when you save the filename  of a post you can write a path that will create a subfolder, like kings/Henry  or queens/Elizabeth hope this helps

**Comment by meehawk** [comment]
I think this issue should be kept open @Enigmatical  since this is still an issue from what i see and am facing the same

**Comment by Remzi1993** [comment]
> I think this issue should be kept open @Enigmatical since this is still an issue from what i see and am facing the same

@meehawk Indeed! I just installed Tina and I get the same issue! This bug is still not fixed.
